corehq/form_processor/abstract_models.py

```diff
@@ -212,7 +212,8 @@ def resolve_case_property(self, property_name):
         case has more than one parent or grandparent.
         """
         result = []
-        self._resolve_case_property(property_name, result)
+        if isinstance(property_name, basestring):
+            self._resolve_case_property(property_name, result)
```

Comment:
there were places in the original code where you could end up passing in None here. for example, places where we get a case property name from a dynamic place like a db model.  it seemed like the right thing to just return an empty result, but i don't have strong feelings. let me know if you'd rather i check for none outside of this call